File: KaloriHesaplama/KaloriHesaplama.java
package kalorihesaplama;

import java.util.Scanner;

public class KaloriHesaplama {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Cinsiyet giriniz: ");
        String cinsiyet = reader.nextLine();
        System.out.print("Adım sayısını giriniz: ");
        int adimSayisi = reader.nextInt();
        int kalori;
        if (cinsiyet.equals("kadin")) {
            kalori = adimSayisi * 30 / 500;
        } else {
            kalori = adimSayisi * 45 / 500;
        }
        System.out.println("Yakılan kalori miktarı:" + kalori);
    }
}

File: KaloriHesaplama/KaloriHesaplama.py
def kalori_hesaplama():

    cinsiyet = input("Cinsiyetinizi girin : ")
    adim_sayisi = int(input("Adım sayısını girin : "))
    kalori = 0

    if cinsiyet.lower() == "kadın":
        kalori = adim_sayisi * 30 / 500
        print("Yakılan kalori miktarı : {}".format(kalori))
    elif cinsiyet.lower() == "erkek":
        kalori = adim_sayisi * 45 / 500
        print("Yakılan kalori miktarı : {}".format(kalori))
    else:
        print("Geçerli bir cinsiyet girilmedi")


kalori_hesaplama()